src/Core/Mesh/DCEL/Face.hpp

```diff
@@ -20,7 +20,8 @@ namespace Core {
 * Chapter 2, Paragraph 2.2, Page 29
 *
 */
-class RA_CORE_API Face : public IndexedObject  {
+
+class [[deprecated]] RA_CORE_API Face : public IndexedObject  {
```

Comment:
Has anyone checked that VS supports the [[deprecated]] flag ? If not it might require a macro in CoreMacros.